ruleset/rules/0800-sysmon_id_1.xml

```diff
@@ -250,4 +250,15 @@
+    <rule id="92110" level="0">
+        <if_group>sysmon_event1</if_group>
+        <field name="win.eventdata.originalFileName" type="pcre2">(?i)PowerShell\.EXE</field>
+        <field name="win.eventdata.commandLine" type="pcre2">(?i)(Remove-Item|ri|rm|rmdir|del|erase|rd)\b</field>
```

Comment:
We still having a limitation for powershell scriptblock, right?

Reply:
Sorry Nico, didn't see this comment. Scriptblock is usable, can be enabled on client side but it's very expensive in log size